media/css/mozorg/contribute/contribute-base.less

```diff
@@ -1007,6 +1007,7 @@ textarea:focus {
     border-collapse: collapse;
     width: 100%;
     margin: 0 auto 40px;
+    font-style: normal;
```

Comment:
Thank you for contributing! I have a couple things to mention before we merge your PR. 

I ran your change locally and it looks like `font-style: normal;` is not quite cascading to the `<address>` tag like intended. 

What we are looking for is the content in the table under "Location" to be "normal"  non-italic text.
<img width="972" alt="Screen Shot 2019-07-15 at 4 10 52 PM" src="https://user-images.githubusercontent.com/1214589/61254996-9b1f8300-a71b-11e9-9b79-6c61e9a480bb.png">


What you could do is select the address tag directly ( from within the `.events-table`) and apply the style. Selecting the specific element you want to make a change to will help in the future if you need apply other font styles to the table as well.

Feel free to comment if you have any questions or have trouble getting it working, I'll be happy to help :)